### PR TITLE
24 remove deprecated packages

### LINKED ISSUE #24: Remove deprecated packages
When running `npm ci` (which is the equivalent of `yarn install --frozen-lockfile`) we get this deprecation warnings

```bash
npm warn deprecated @types/wordpress__api-fetch@3.23.1: This is a stub types definition. @wordpress/api-fetch provides its own type definitions, so you do not need this installed.
npm warn deprecated @types/wordpress__blob@2.8.0: This is a stub types definition. wordpress__blob provides its own type definitions, so you do not need this installed.
npm warn deprecated @types/wordpress__dom-ready@2.9.0: This is a stub types definition. wordpress__dom-ready provides its own type definitions, so you do not need this installed.
npm warn deprecated @types/swiper@6.0.0: This is a stub types definition. swiper provides its own type definitions, so you do not need this installed.
npm warn deprecated rimraf@2.6.3: Rimraf versions prior to v4 are no longer supported
npm warn deprecated rimraf@2.7.1: Rimraf versions prior to v4 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
npm warn deprecated mkdirp@0.3.0: Legacy versions of mkdirp are no longer supported. Please update to mkdirp 1.x. (Note that the API surface has changed to use Promises in 1.x.)
npm warn deprecated react-instantsearch-hooks@6.47.3: package has moved to react-instantsearch-core
npm warn deprecated react-instantsearch-hooks-web@6.47.3: package has moved to react-instantsearch
```

### PR DESCRIPTION
**Pull request by kuuak**
fixes #24 + fix dependencies vulnerabilities from 16 to 0

⚠️ Merge #33 before this one

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,11 +38,9 @@
     "@types/react": "^18",
     "@types/react-dom": "^18",
     "@types/sprintf-js": "^1.1.4",
-    "@types/wordpress__api-fetch": "^3.23.1",
-    "@types/wordpress__blob": "^2.8.0",
     "@types/wordpress__block-editor": "^11.5.15",
     "@types/wordpress__blocks": "^12.5.13",
-    "@types/wordpress__dom-ready": "^2.9.0",
+    "@wordpress/api-fetch": "^7.4.0",
```
Comment (charl0tee) [inline package.json:43]: @kuuak But this means that we import `@wordpress` package inside our final JS on the frontend no ? I don't know if we can find an alternative to this ?

**Review by charl0tee — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (kuuak) [reply package.json:43]: @charl0tee It's a devDependency therfore it should not be included in the final JS. I will validate and check to be 100% sure about that

Comment (kuuak) [reply package.json:43]: @charl0tee I can validate that it doesn't leak to the client JS

|   | before packages changes | after |
| --- | --- | --- |
| Nextjs  | Capture d’écran 2024-07-29 à 16 43 29 | Capture d’écran 2024-07-29 à 16 51 46 |
| WordPress editor | Capture d’écran 2024-07-29 à 16 46 55 | Capture d’écran 2024-07-29 à 16 52 58 |

Comment (charl0tee) [reply package.json:43]: @kuuak Aaah, nice ! 👌🏽

**Review by charl0tee — DISMISSED** [review]

**Pull request merged**